src/khoj/routers/api_chat.py

```diff
@@ -528,6 +528,20 @@ async def send_rate_limit_message(message: str):
             await conversation_command_rate_limiter.update_and_check_if_valid(websocket, cmd)
             q = q.replace(f"/{cmd.value}", "").strip()
 
+        custom_filters = ""
```

Comment:
Based on above comment, would want to change to something like 
`custom_filters = []`

and then later, 

`custom_filters.append("site:khoj.dev")`.